pkg/plugin/driver/base.go

```diff
@@ -66,6 +66,11 @@ func (b *Base) GetName() string {
 	return b.Definition.Name
 }
 
+// GetIOrder returns the order of this Job plugin.
+func (b *Base) GetOrder() int {
+	return b.Definition.Order
+}
+
 // GetSecretName gets a name for a secret based on the plugin name and session ID.
 func (b *Base) GetSecretName() string {
```

Comment:
Hi @johnSchnake,

Consider rewriting the function signature as func `(b Base) GetSecretName() string` since there is no reason for the function to be able to write any values to the address of `b`. A read on a copy will suffice and offers better protection.